### PR TITLE
Document additional  Project Public API workflows

### PR DESCRIPTION
**Pull request by pmatseykanets**
## Description

Document additional Public API workflows for the Project resource.

### CODE REVIEW

**Review by crobby — DISMISSED** [review]

**Review by crobby — APPROVED** [review]

**Review by olblak — APPROVED** [review]

**Comment by olblak** [comment]
Apparently @btat or @martyav need to review this pullrequest and only then they can merge it

File: docs/api/workflows/projects.md [hunk 1]
```diff
@@ -48,6 +48,9 @@ EOF
 ```
 
 Setting the `field.cattle.io/creatorId` field allows the cluster member account to see project resources with the `get` command and view the project in the Rancher UI. Cluster owner and admin accounts don't need to set this annotation to perform these tasks.
+Setting the  `field.cattle.io/creator-principal-name` annotation to the user's principal preserves it in a projectroletemplatebinding automatically created for the project owner.
```
Comment (btat) [inline docs/api/workflows/projects.md:51]:
```suggestion

Setting the  `field.cattle.io/creator-principal-name` annotation to the user's principal preserves it in a projectroletemplatebinding automatically created for the project owner.
```

Adding newline for consistency with the rest of the doc. Right now line 50 and 51 renders as continuous line.

File: versioned_docs/version-2.10/api/workflows/projects.md [hunk 2]
```diff
@@ -48,6 +48,9 @@ EOF
 ```
 
 Setting the `field.cattle.io/creatorId` field allows the cluster member account to see project resources with the `get` command and view the project in the Rancher UI. Cluster owner and admin accounts don't need to set this annotation to perform these tasks.
+Setting the  `field.cattle.io/creator-principal-name` annotation to the user's principal preserves it in a projectroletemplatebinding automatically created for the project owner.
```
Comment (btat) [inline versioned_docs/version-2.10/api/workflows/projects.md:51]:
```suggestion

Setting the  `field.cattle.io/creator-principal-name` annotation to the user's principal preserves it in a projectroletemplatebinding automatically created for the project owner.
```

**Review by btat — CHANGES_REQUESTED** [review]

**Comment by pmatseykanets** [comment]
@btat Should this PR be opened against v2.10.0 or main branch?

**Comment by btat** [comment]
@pmatseykanets v2.10.0 is more suitable in this case since there are 2.10 specific changes to `/docs`, which we don't want to publish yet. The `/docs/` folder represents the latest released version, which is currently 2.9 and will represent 2.10 upon its release.

**Review by btat — APPROVED** [review]

**Pull request merged**